javascript/statements.json

```
@@ -1499,6 +1487,84 @@
               }
             }
           }
+        },
+        "service_worker_support": {
+          "__compat": {
+            "description": "Available in service workers",
+            "support": {
+              "chrome": {
+                "version_added": "91"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "114"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "15"
```

Can't find evidence of support in Safari:
```suggestion
                "version_added": false
```